chart/templates/configmap-env.yaml

```diff
@@ -7,12 +7,13 @@ metadata:
 data:
   {{- if .Values.postgresql.enabled }}
   DB_HOST: {{ template "mastodon.postgresql.fullname" . }}
+  DB_PORT: "5432"
   {{- else }}
   DB_HOST: {{ .Values.postgresql.postgresqlHostname }}
+  DB_PORT: {{ .Values.postgresql.postgresqlPort }}
```

Comment:
It would be nice to default this to 5432 so that users don't have to specify a port just because they're using an external host.

Reply:
Good point, just added it as a default 👍